website/content/sdk/guides/intros/stocks/stocks-screener.md

```diff
@@ -0,0 +1,314 @@
+---
+title: Stocks Screener
+---
+
+The Stocks Screener module imports the same screener found in the OpenBB Terminal, and adds advanced scripting capabilities in the SDK layer. The screener utilizes presets (.ini files) to scan for stocks meeting the criteria defined. User-generated files are kept in the `OpenBBUserData` folder under, `~/OpenBBUserData/presets/stocks/screener/`. There are also presets included within the cloned repo folder: 
+
+`path_to_git_clone/OpenBBTerminal/openbb_terminal/stocks/screener/presets`
+
+For convenience, they are supplied here as a ZIP file, [screener_presets.zip](https://github.com/OpenBB-finance/OpenBBTerminal/files/10119826/screener_presets.zip).
+
+This screener can find stocks on three exchanges:
+
+- AMEX
+- NASDAQ
+- NYSE
+
+The data returned is grouped into five categories:
+
+- Overview
+- Ownership
+- Performance
+- Technical
+- Valuation
+
+## How to Use
+
+### Template.ini
+
+This file is for seeing all possible arguments and parameters for the screener. Reference this file (included in the ZIP file above) for the master list of settings. There are four categories of parameters to set. Without any parameters, a screener preset file must contain the following at a minimum:
+
+```console
+# Author of preset: 
+# Description: 
+
+[General]
+
+[Descriptive]
+
+[Fundamental]
+
+[Technical]
+```
+
+Parameters should be added as required, and they all have pre-defined values; for example, Price/Free Cash Flow:
+
+```console
+# Any, Low (<15), High (>50), Under 5, Under 10, Under 15, Under 20, Under 25, Under 30, Under 35, Under 40, Under 45, Under 50, Under 60, Under 70, Under 80, Under 90, Under 100, Over 5, Over 10, Over 15, Over 20, Over 25, Over 30, Over 35, Over 40, Over 45, Over 50, Over 60, Over 70, Over 80, Over 90, Over 100
+
+Price/Free Cash Flow = Any
+```
+
+## Examples
+
+To make a screener with only a few filters, it should look something like this:
+
+```console
+# Author of preset: Danglewood
```

Comment:
Let's change this OpenBB

Reply:
Done. ZIP file updated to reflect this as well.